src/main/java/graphql/annotations/connection/simple/SimpleGraphQLConnection.java

```diff
@@ -0,0 +1,14 @@
+package graphql.annotations.connection.simple;
+
+import graphql.annotations.connection.GraphQLConnection;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ElementType.FIELD, ElementType.METHOD})
+@Retention(RetentionPolicy.RUNTIME)
+@GraphQLConnection(connectionFetcher = SimplePaginatedDataConnectionFetcher.class, validator = SimplePaginatedDataConnectionTypeValidator.class, connectionType = SimpleRelay.class)
+public @interface SimpleGraphQLConnection {
+}
```

Comment:
Nice addition, makes things even more simple - can you update the unit test to use this annotation ?

Reply:
Done